CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [v0.10.1](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.10.1) (July 09, 2021)
+
+[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.10.0...v0.10.1)
+
+**Breaking changes:**
+
+- Renamed AdvancedRegionSpec to AdvancedReplicationSpec [\#230](https://github.com/mongodb/go-client-mongodb-atlas/pull/230) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

Comment:
this line is from the previous release

Reply:
Should I remove it manually?

Reply:
if it was auto aded then let's keep it, maybe the tool keeps the breaking message for minor patches